src/EditorFeatures/Test2/IntelliSense/VisualBasicCompletionCommandHandlerTests.vb

```diff
@@ -444,7 +453,8 @@ End Class
             End Using
         End Function
 
-        <MemberData(NameOf(AllCompletionImplementations))> <WpfTheory, Trait(Traits.Feature, Traits.Features.Completion)>
+        <MemberData(NameOf(AllCompletionImplementations))>
+        <WpfTheory, Trait(Traits.Feature, Traits.Features.Completion)>
         Public Async Function TestNavigationBeforeCompletedComputation(completionImplementation As CompletionImplementation) As Task
```

Comment:
❗️ Add a copy of this test that uses `SendMoveToNextCharacter` instead of `SendMoveToPreviousCharacter`. For this situation, the completion session should *not* be visible.

Reply:
[https://github.com/dotnet/roslyn/issues/30970](https://github.com/dotnet/roslyn/issues/30970)

---
In reply to: [229268810](https://github.com/dotnet/roslyn/pull/29016#discussion_r229268810) [](ancestors = 229268810)